### PR TITLE
homework

### PR DESCRIPTION
**Pull request by AlexanderVVasilenko**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        # Sort the list based on the position and reconstruct the word
+        reversed_word = ''.join(symbol[1] for symbol in sorted(symbols, key=lambda x: x[0]))
+        reversed_words.append(reversed_word)
+
+    # Join the reversed words to form the final result
+    result = ' '.join(reversed_words)
+    return result
+
+
+# Example usage
```
Comment (levovit) [inline main.py]: these comments are stupid.

"the perfect code has no comments" (c) 

It is obvious that this is example of usage, you don't need to specify that

File: test_reverse_words.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# test_reverse_words.py
```
Comment (levovit) [inline test_reverse_words.py]: what is this? remove!!!!!!!!!!!!!!!!!!!!!!

File: test_reverse_words.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+from main import reverse_words  # Replace 'your_module' with the actual module name
+
+
+class TestReverseWords:
```
Comment (levovit) [inline test_reverse_words.py:4]: How to run tests? 

Create a Readme.md file with an explanation how to run these tests

Comment (levovit) [reply test_reverse_words.py:4]: Is this pytest? Add requirements.txt with all libs needed. Add to Readme.md an explanation of how to set up virtuaenv, and install all requirements. and how to run your program

**Review by levovit — CHANGES_REQUESTED** [review]
Tests are greatly written, implementation of anagram is good. 

But everything else is bad. Rework needed

**Comment by AlexanderVVasilenko** [comment]
> Tests are greatly written, implementation of anagram is good.
> 
> But everything else is bad. Rework needed

done

**Pull request merged**